### ISSUE TITLE
Change AMP consent preferences 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#8750]
**Is your feature request related to a problem? Please describe.**
From NEWSWORLDSERVICE-1215

We  must ensure users can change their consent preferences on AMP pages. 

User will be able to change their AMP consent settings by clicking the "Ad choices / Do not sell my info" link in the footer on AMP pages

**Scenario: Footer Link on AMP** 

- Given I have already accepted consent on AMP pages
- When I click on the "ad / Choices do not sell my info link" in the footer on AMP pages
- then the extended AMP banner is displayed to
- And I can choose to accept or reject cookies  

<br>

- Given I am a screenreader user on a Simorgh AMP page
- when I click on “ad / Choices do not sell my info” in the footer  
- then it is announced as button

 **Scenario: Footer Link on canonical pages (no changes)** 

- Given I am on a Simorgh canonical page
- when I click on the "ad / Choices do not sell my info link" in the footer
- then I am redirected to https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/
- (i.e. current behaviour is unchanged)

<br>

- Given I am a screenreader user on a Simorgh canonical page
- when I click on “ad / Choices do not sell my info” in the footer  
- then it is announced as a link 

**Describe alternatives you've considered**
- Implement logic show or hide the banner
- When 'ad choices / do not sell my info' is clicked change the state as below showing the manage settings page:
```
tap:AMP.setState({ isManagingSettings: true })
```

**Testing notes**
- Add E2E test for manageSettings flow (cypress)
- Jest unit tests

- [x] This feature is expected to need manual testing.

**Additional context**

**Comment by andrewscfc** [comment]
Blocked by https://github.com/bbc/simorgh/issues/8668

**Comment by aeroplanejane** [comment]
Taking this out  and adding here: https://github.com/bbc/simorgh/issues/8895
**Scenario: Focus Management**

- Given I have activated the “Manage my settings” button on the consent banner  
- When the expanded consent banner is displayed
- Then focus should be on the H2 at the top of banner e.g. (“Manage consent settings in AMP”)

<br>

- Given I am on the consent banner 
- When I select to accept or reject data collection 
- Then the consent banner is closed  
- And my focus is returned to the last element that previously had focus on the page (ie. if the “ad choices / do not sell my info” link in the footer was my last focused element, then my focus should return here)
~~
